osu.Game/Screens/OsuScreen.cs

```diff
@@ -128,5 +147,33 @@ protected override bool OnExiting(Screen next)
             Beatmap.UnbindAll();
             return false;
         }
+
+        private void logoSetup(bool resuming) => LogoSetup(logo, resuming);
+
+        protected virtual void LogoSetup(OsuLogo logo, bool resuming)
+        {
+            logo.Action = null;
+            logo.FadeOut(300, Easing.OutQuint);
+        }
+
+        private void logoOnExiting()
+        {
+            logo.ClearTransforms();
+            LogoOnExiting(logo);
+        }
+
+        protected virtual void LogoOnExiting(OsuLogo logo)

```

Comment:
How about combining `LogoOnExiting` and `LogoOnSuspending` into `RemoveLogo(OsuLogo logo, bool suspending)`, following `LogoSetup`'s signature?

Reply:
Otherwise would prefer renaming these to `RemoveLogo` and `SuspendLogo`.

Reply:
Can't be combined as they serve different purposes.

Remove doesn't make sense because the logo isn't being removed